src/nebula/Job/Implementation/DefaultJobRunnerManager.cs

```diff
@@ -92,6 +92,11 @@ public bool IsJobRunnerActive(string jobId)
             return _runners.TryGetValue(jobId, out var runner) && !runner.IsProcessTerminated;
         }
 
+        public bool IsJobRunnerStarted(string jobId)
+        {
+            return _runners.TryGetValue(jobId, out var runner) && !runner.IsProcessRunning;
```

Comment:
Why "NOT runner.IsProcessRunning"? I mean, why negate?
Is it consistent with the name of the method?